<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameObjectMaterialComponentsWidgetQt</class>
 <widget class="QWidget" name="GameObjectMaterialComponentsWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>352</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Material Game Object Components</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget, QDialog, QMenuBar::item
{
	background-color: rgb(50, 50, 50);
	color: rgb(255, 255, 255);
}

QHeaderView::section
{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(75, 75, 75, 255));
	padding-left: 4px;
	border: 1px solid rgb(110, 110, 110);
}

QMenuBar::item:selected, QMenu::item:selected
{
    background-color: rgb(120, 120, 120);
	border: 1px solid rgb(200, 200, 200);
}

QPushButton:hover
{
    background-color: rgb(75, 75, 75);
}

QPushButton:pressed
{
    background-color: rgb(100, 100, 100);
}

QLineEdit
{
	border-width: 2px;
	border-style: inset;
	border-color: rgb(60, 60, 60);
	border-radius: 5px;
	padding: 1px;
}

QLineEdit:focus
{
	background-color: rgb(75, 75, 75);
}

QDockWidget::title
{
	background-color: rgb(65, 65, 65);
	padding: 5px;
}

QTabBar::tab 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(80, 80, 80, 255), stop:1 rgba(60, 60, 60, 255));
	padding: 5px;
	borde-width: 3px; 
	border-color: rgb(110, 110, 110);
	border-style: inset;
	border-radius: 8px;
	font-size: 11px;
}

QTabBar::tab:selected 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(120, 120, 120, 255), stop:1 rgba(60, 60, 60, 255));
	border-style: outset;
    border-bottom-color: rgb(60, 60, 60);
}

QTreeWidget
{
	alternate-background-color: rgb(60, 60, 60);
}

QTreeWidget::branch:has-children
{
	image: url(:/XEngine_Editor/Resources/Images/branch-closegrey.png) 0;
}

QTreeWidget::branch:has-children:open
{
	image: url(:/XEngine_Editor/Resources/Images/branch-opengrey.png) 0;
}

QTreeWidget::branch:has-children:open:selected
{
	image: url(:/XEngine_Editor/Resources/Images/branch-openblue.png) 0;
}

QTreeWidget::branch:has-children:!open:selected
{
	image: url(:/XEngine_Editor/Resources/Images/branch-closeblue.png) 0;
}
</string>
  </property>
  <widget class="QTreeWidget" name="m_MaterialsTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="rootIsDecorated">
    <bool>true</bool>
   </property>
   <property name="itemsExpandable">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Materials</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_AddMaterialButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Material</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_RemoveMaterialButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Material</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_MoveMaterialDownButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>\/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_MoveMaterialUpButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>/\</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_VSClearAssetButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_CSAddAssetButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>260</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_CSClearAssetButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelVSAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Vertex Shader</string>
   </property>
   <property name="text">
    <string>VS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_VSPropsButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set Variables</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelGSAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Geometry Shader</string>
   </property>
   <property name="text">
    <string>GS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_DSAddAssetButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>290</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_HSAddAssetButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>320</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_PSAssetTextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pixel Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_HSAssetTextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Hull Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_GSAddAssetButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>230</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_VSAddAssetButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>170</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PSAddAssetButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>200</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_DSPropsButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set Variables</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_VSAssetTextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Vertex Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_HSClearAssetButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelCSAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Compute Shader</string>
   </property>
   <property name="text">
    <string>CS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelDSAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Domain Shader</string>
   </property>
   <property name="text">
    <string>DS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_CSPropsButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set Variables</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelHSAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Hull Shader</string>
   </property>
   <property name="text">
    <string>HS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_DSClearAssetButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PSPropsButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set Variables</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_CSAssetTextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Compute Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_GSAssetTextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Geometry Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_HSPropsButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set Variables</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelPSAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pixel Shader</string>
   </property>
   <property name="text">
    <string>PS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_GSPropsButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set Variables</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_GSClearAssetButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_PSClearAssetButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_DSAssetTextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Domain Shader</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelMaterialProperties">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>144</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Material Properties:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
